### DISCUSSION TITLE
in vb.net windows form standalone application multiple instance or another application with cefsharp is open cefsharp is not intialising

### DISCUSSION THREAD

**Discussion by chandra-dd** [cefsharp/CefSharp#4973] [General]
in vb.net windows form standalone application multiple instance or another application with cefsharp is open cefsharp is not initialized and  check the log file  where to see how to run multiple vb.net applications embed with cefsharp ,at a time in the same machine .

**Comment by amaitland** [comment]
You need to set a `RootCachePath` unique to your application. Also multiple processes accessing the `RootCachePath` at the same time is now supported.

https://github.com/cefsharp/CefSharp/blob/cd0d144c5064e1c802dec861f5c715f416d92763/CefSharp.Core/CefSettingsBase.cs#L148
